boilerplate/app/components/Toggle.tsx

```diff
@@ -262,26 +262,26 @@ function Checkbox(props: ToggleInputProps) {
     disabled && colors.palette.neutral400,
     status === "error" && colors.errorBackground,
     colors.palette.neutral200,
-  ].filter((item): item is string => Boolean(item))[0]
+  ].filter(Boolean)[0]
```

Comment:
If we're looking for the first "truthy" value, we could call `find(Boolean)` so that we don't loop through the whole array; however, we'd have to extend `find` the same way we did `filter` for typing.